### PR TITLE
docs(docs): add explanation for serving multiple projects

### PR DESCRIPTION
**Pull request by arrenv**
_[Please make sure you have read the submission guidelines before posting an PR](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#submit-pr)_

> _Please make sure that your commit message follows our format._

> Example: `fix(nx): must begin with lowercase`

### CODE REVIEW

**Comment by isaacplmann** [comment]
I believe @FrozenPandaz intended this sentence to mean run those two commands in separate terminal windows.

Your modification would work, but might be confusing to someone who is just learning nx.  @FrozenPandaz, what do you think?

**Comment by FrozenPandaz** [comment]
I agree, it should be in separate terminals to keep it as beginner-friendly as possible.

Can you please add some text that clarifies that it should be in separate terminals?

Also, please make the same change in the react docs as well.

**Comment by vsavkin** [comment]
@arrenv could you make the changes Jason suggested?

File: docs/angular/tutorial/06-proxy.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ It created a proxy configuration that allows the Angular application to talk to
 This configuration tells `nx serve` to forward all requests starting with `/api` to the process listening on port 3333.
 
 !!!!!
-Now run both "nx serve todos" and "nx serve api", open http://localhost:4200. What do you see?
+Now run both "todos" and "api", using "nx run-many --parallel --target=serve --projects=todos,api --maxParallel=100" open http://localhost:4200. What do you see?
```
Comment (arrenv) [inline docs/angular/tutorial/06-proxy.md:42]:
```suggestion
Now run both "nx serve todos" and "nx serve api" in separate terminals, open http://localhost:4200. What do you see
```

**Review by arrenv — COMMENTED** [review]
Update based on commit feedback.

**Comment by FrozenPandaz** [comment]
Thank you for your contribution!

In the future, please adhere to the commit message format in the commits as well. :smile:

**Pull request merged**